src/w2grid.js

```diff
@@ -146,6 +146,7 @@
         this.columns      = [];       // { field, caption, size, attr, render, hidden, gridMinWidth, editable }
         this.columnGroups = [];       // { span: int, caption: 'string', master: true/false }
         this.records      = [];       // { recid: int(requied), field1: 'value1', ... fieldN: 'valueN', style: 'string',  changes: object }
+		this.bufrec       = [];       // AP19 aleady buffered range...
```

Comment:
You use tabs for code alignment, you should use spaces, as elsewhere in the project.